guard/tests/utils.rs

```diff
@@ -64,6 +65,37 @@ pub fn get_full_path_for_resource_file(path: &str) -> String {
     return resource.display().to_string();
 }
 
+pub fn replace_home_directory_with_tilde(text: String) -> String {
+    let home_dir_string = match env::var("HOME") {
```

Comment:
nit: should we handle windows to such as "HOMEPATH"?

Reply:
Resolved offline